spec/schemas/sql._common.yaml

```diff
@@ -85,4 +85,59 @@ components:
         user:
           type: object
         execution_time:
-          type: object
\ No newline at end of file
+          type: object
+    SqlSettings:
+      type: object
+      properties:
+        transient:
+          $ref: '#/components/schemas/Transient'
+    SqlSettingsResponse:
+      type: object
+      properties:
+        acknowledged:
+          type: boolean
+        persistent:
+          type: object
+        transient:
+          $ref: '#/components/schemas/Transient'
+    Transient:
+      type: object
+      properties:
+        plugins:
+          $ref: '#/components/schemas/Plugins'
+    Plugins:
+      type: object
+      properties:
+        ppl:
+          $ref: '#/components/schemas/Ppl'
+        query:
+          $ref: '#/components/schemas/Query'
+        sql:
+          $ref: '#/components/schemas/Sql'
+    Ppl:
+      type: object
+      properties:
+        enabled:
+          type: string
+    Query:
+      type: object
+      properties:
+        memory_limit:
+          type: string
+        size_limit:
+          type: string
```

Comment:
Double-checking that those are actually strings?

Reply:
This API accept string and also numbers. 
<img width="1028" alt="image" src="https://github.com/user-attachments/assets/26753831-f572-41c6-af4f-224431be5405">
<img width="1096" alt="image" src="https://github.com/user-attachments/assets/fe45de87-58fe-483f-977c-2078f0770002">